src/pages/documentation/release-notes/2020.md

```diff
@@ -316,19 +194,10 @@ The following updates were made in this release:
 ### Mobile Core tag Extension 2.1.15
 
 * Added Modify Data action for tag Rules. See more information on usage here: [Modify data in SDK events](../user-guides/modify-data.md).
-* Update requires Mobile Core iOS minimum version 2.7.2 and Android minimum version 1.5.5
```

Comment:
this refers to the tag extension dependencies, I suggest we keep the Android min version